hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/RocksDBStoreMetrics.java

```diff
@@ -105,6 +105,8 @@ public class RocksDBStoreMetrics implements MetricsSource {
   private static final String NUM_FILES_AT_LEVEL = "num_files_at_level";
   private static final String SIZE_AT_LEVEL = "size_at_level";
 
+  private static final String LAST_SEQUENCE_NUMBER = "last_sequence_number";
```

Comment:
Looks straightforward change introducing  LAST_SEQUENCE_NUMBER metrics. On similar lines as HistogramData,TickerTypeData,PropertyData we have introduced getLatestSequenceNumber and updating metrics with calling  getLatestSequenceNumber via getMetrics.

Reply:
@mohammadjkhan @devmadhuu Please Review as metrics related